src/pages/api/matching.ts

```diff
@@ -22,6 +29,8 @@ interface MatchingBaseData {
     nocCode: string;
     operatorShortName: string;
     serviceDescription: string;
+    email: string;
+    uuid: string;
```

Comment:
If we've removed the uuid from the other matching data, shall we remove it here too?

Reply:
I think this also propagates through this file down on lines 83, 93, 103 and 162